packages/camera/camera_platform_interface/lib/src/types/video_stabilization_mode.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The possible video stabilization modes that can be capturing video.
+enum VideoStabilizationMode {
+  /// Video stabilization is disabled.
+  off,
+
+  /// Basic video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_ON on Android
+  /// and throws CameraException on iOS.
+  on,
+
+  /// Standard video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_PREVIEW_STABILIZATION on Android
+  /// (camera_android_camerax) and to AVCaptureVideoStabilizationModeStandard
+  /// on iOS.
+  standard,
+
+  /// Cinematic video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_PREVIEW_STABILIZATION on Android
+  /// (camera_android_camerax) and to AVCaptureVideoStabilizationModeCinematic
+  /// on iOS.
+  cinematic,
+
+  /// Extended cinematic video stabilization is enabled.
+  ///
+  /// Maps to AVCaptureVideoStabilizationModeCinematicExtended on iOS and
+  /// throws CameraException on Android.
```

Comment:
If next year, Android gains the ability to support extended cinematic stabilization, would adding support for it be a bug?

Reply:
OK, that answers my question, so I will remove that comment.